eth/stagedsync/stage_execute.go

```diff
@@ -129,6 +130,11 @@ func executeBlock(
 		if err = rawdb.AppendReceipts(tx, blockNum, receipts); err != nil {
 			return err
 		}
+
+		if stateSyncReceipt != nil {
+			rawdb.WriteBorReceipt(tx, block.Hash(), block.NumberU64(), stateSyncReceipt)
+			rawdb.WriteBorTxLookupEntry(tx, block.Hash(), block.NumberU64())
```

Comment:
If you have table with hashed keys, and can't Append it - then better write it to `batch` instead of `tx`.